css/main-styling.css

```diff
@@ -0,0 +1,89 @@
+body {
+  font-family: 'Roboto Slab', serif;
+  font-size: 18px;
+  line-height: 1.5;
+  cursor: default;
+  font-color: black;
+}
+
+.wrapper {
+  max-width: 1280px;
+  margin: auto;
+}
+
+h1 {
+  font-size: 2em;
+  line-height: 1;
+}
+
+h3 {
+  font-size: 1.25em;
+  font-weight: bold;
+}
+
+p {
+  margin: 0 0 1.5em 0;
+  font-size: 14px;
+}
+
+.button {
+  text-decoration: none;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  font-weight: bold;
+  color: inherit;
+  display: block;
+  text-align: center;
+  box-shadow: inset 0 -3px 0 rgba(0,0,0,.2);
+  border-radius: 5px;
+}
+
+#page-header {
+  background: #97c0e6;
+  overflow: hidden;
+  padding: 0 3%;
+}
+
+/*#page-header h1 {
+  insert img logo
+}*/
+
+#page-header a {
+  color: white;
+  text-decoration: none;
+}
+
+#page-header nav {
+  float: right;
+}
+
+#page-header nav li {
+  float: left;
+}
+
+#page-header nav a {
+  display: block;
+  /*double check display after logo is input*/
+  padding: 20px 15px;
+}
+
+#page-header nav li a:hover {
+  color: inherit;
+  font-weight: bold;
+}
+
+#page-header nav strong {
+  text-transform: uppercase;
+}
+
+.buttons {
+  overflow: hidden;
+  padding: 2% 0;
+}
+
+.actions {
+  float: left;
+  width: 27%;
```

Comment:
Since you're using your class name to set up columns, it might be useful to name your columns something like `col-1-3` to represent columns that are 1/3 the width of their container. Not necessary, just an idea.